setup.py

```diff
@@ -5,7 +5,7 @@
 from setuptools import setup
 
 ROOT_DIR = Path(os.path.join(os.path.dirname(__file__)))
-PACKAGE_VERSION = f'{0}.{1}.{0}'
+PACKAGE_VERSION = f'{0}.{1}.{1}'
```

Comment:
I think the package version should be 1.0.0 since this changes is breaking the backward compatibility (a major update)

Reply:
You are right, I will change the version on my next commit.